src/index.html

```diff
@@ -1,12 +1,348 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="/images/logo.png" alt="logo"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="products">
+          <a href="#" class="products__link is-active">Apple</a>
+          <a href="#" class="products__link">Samsung</a>
+          <a href="#" class="products__link">Smartphones</a>
+          <a
+            href="#" class="products__link"
+            data-qa="nav-hover"
+          >
+            Laptops &amp Computers
+          </a>
+          <a href="#" class="products__link">Gadgets</a>
+          <a href="#" class="products__link">Tablets</a>
+          <a href="#" class="products__link">Photo</a>
+          <a href="#" class="products__link">Video</a>
+        </ul>
+      </nav>
+    </header>
+    <main class="content">
+      <div class="card" data-qa="card">
+        <img
+          src="../src/images/imac.jpeg" alt="APPLE A1419 iMac 27 Retina 5K Monoblock&quot (MNED2UA/A)"
+          class="card__photo"
+        >
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rating-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a
+          ref="#buy" class="card__buy"
+          data-qa="hover"
+        >
+        Buy
+        </a>
+      </div>
+      <div class="card" data-qa="card">
+        <img
+          src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27
+        Retina 5K Monoblock&quot (MNED2UA/A)"
+          class="card__photo"
+        >
```

Comment:
Fix all similar cases
```suggestion
        <img
          src="../src/images/imac.jpeg"
          alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
          class="card__photo"
        >
```